mtags/src/main/scala-3/scala/meta/internal/pc/completions/AmmoniteFileCompletions.scala

```diff
@@ -0,0 +1,127 @@
+package scala.meta.internal.pc
+package completions
+
+import java.nio.file.Path
+import java.{util as ju}
+
+import scala.collection.JavaConverters.*
+
+import scala.meta.internal.mtags.MtagsEnrichments.*
+import scala.meta.internal.pc.AutoImports.AutoImport
+import scala.meta.internal.pc.AutoImports.AutoImportsGenerator
+import scala.meta.internal.pc.printer.MetalsPrinter
+import scala.meta.io.AbsolutePath
+import scala.meta.pc.OffsetParams
+import scala.meta.pc.PresentationCompilerConfig
+import scala.meta.pc.PresentationCompilerConfig.OverrideDefFormat
+import scala.meta.pc.SymbolSearch
+
+import dotty.tools.dotc.ast.tpd.Tree
+import dotty.tools.dotc.ast.tpd.*
+import dotty.tools.dotc.ast.untpd.ImportSelector
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Definitions
+import dotty.tools.dotc.core.Flags
+import dotty.tools.dotc.core.Flags.*
+import dotty.tools.dotc.core.NameKinds.DefaultGetterName
+import dotty.tools.dotc.core.NameKinds.NameKind
+import dotty.tools.dotc.core.Names.Name
+import dotty.tools.dotc.core.StdNames
+import dotty.tools.dotc.core.SymDenotations.SymDenotation
+import dotty.tools.dotc.core.Symbols.NoSymbol
+import dotty.tools.dotc.core.Symbols.Symbol
+import dotty.tools.dotc.core.Types.Type
+import dotty.tools.dotc.interactive.Interactive
+import dotty.tools.dotc.interactive.InteractiveDriver
+import dotty.tools.dotc.util.SourceFile
+import dotty.tools.dotc.util.SourcePosition
+import dotty.tools.dotc.util.Spans.Span
+import org.eclipse.{lsp4j as l}
+
+object AmmoniteFileCompletions:
+  val CURSOR = "_CURSOR_"
```

Comment:
```suggestion
  val CURSOR = "CURSOR"
```
It's adifferent name in Scala 3. Probably best to move it to a separate object. It's defined in CompletionsProvider:145